pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -613,16 +613,27 @@ func expandDiskImageOffline(imagePath string, size int64) error {
 
 func possibleGuestSize(disk api.Disk) (int64, bool) {
 	var err error
-	capacityResource := disk.Capacity
-	if capacityResource == nil {
+	storageCapacityResource := disk.StorageCapacity
```

Comment:
Not sure if it is cleaner to do min of quantities first, and then return int64 or switch to int64 first and then do min like here.

Reply:
I've done something like this - disk.Capacity is now *int64